src/extra/python/scripts/era5_invariant_fields/README.md

```diff
@@ -0,0 +1,11 @@
+# Topography Source Files
+## ERA-5
+
+Isca provides a number of preconstructed topography files for use with common resolutions, which can be found under `Isca/input/era5_smoothed_topography_land_masks`. These were constructed with the script `Isca/src/extra/python/scripts/create_era5_topography.py` and make use of the ERA-5 invariant fields as an input. These files are not included as part of Isca, however they may be obtained by using either directly from the [CEDA](https://catalogue.ceda.ac.uk/uuid/2c8f38fac04945b89cf12d6e9c928c6f/) archive (recommended) or from the ERA-5 [CDS](https://cds.climate.copernicus.eu/datasets). Note that a login is required for both websites.

```

Comment:
This looks perfect. Only one small suggested tweak, which would be to swap `These files are not included...` to `The ERA-5 invariant fields are not included...`.